sos/plugins/ovn_host.py

```diff
@@ -40,6 +42,11 @@ def setup(self):
 
         self.add_journal(units="ovn-controller")
 
+    def check_enabled(self):
+        return (any([os.path.isfile(
+            os.path.join(pp, pidfile)) for pp in pid_paths]) or
+            super(OVNHost, self).check_enabled())
+
```

Comment:
This doesn't actually check that the `ovn-controller` process is running - it just checks to see if any possible PID file exists. For that you can just use the regular `files = ` member of the `Plugin` class - there's no need to override `setup()`.